src/notifications/screens/notifications.screen.js

```diff
@@ -346,6 +346,14 @@ class Notifications extends Component {
     const { markAsRead, navigation } = this.props;
 
     markAsRead(notification.id);
+
+    if (
+      notification.subject.type !== 'PullRequest' &&
+      notification.subject.type !== 'Issue'
+    ) {
+      return;
+    }
```

Comment:
How about to check the opposite condition? In this case we can add or remove this exceptions in the future.
Also could be worth move `SubjectType` to an other file or export it to be imported in `notifications.screen.js`.
```js
if (notification.subject.type === SubjectType.release) {
  return;
}
```

Reply:
@ocarreterom checking the opposite means a potential crash at runtime :/
Good idea for SubjectType, but where would you put this enum? 🤔

Reply:
We have the problem that the project structure for components' directory is too flat.
Maybe each component should have their own directory: e.g `/components/notification/notification-list-item.js`.
Then, the enum could be in `/components/notification/notification-subject-type.js`.

The quick solution is to export that enum from `notification-list-item.component.js` waiting for a structure refactor. 😞

Reply:
In a perfect world, these definitions would be part of the API definition, not inside components.. 💭

Reply:
I'm agree. 👍